### DISCUSSION TITLE
Warten am nächsten trigger

### DISCUSSION THREAD

**Discussion by acton59** [Courseplay/courseplay#7416] [General]
Hallo,

wo versteckt sich die Einstellung "warten am nächsten Trigger, damit der Abfahrer bei der letzten Tour nicht nochmal auf das
Feld fährt.
Habe ich früher sehr oft benutzt.

**Comment by Tensuko** [comment]
Haben wir mit absicht raus genommen, da es all zu oft vergessen wurde richtig um zu stellen.

Außerdem bedarf die Option der gleichen Aufmerksamkeit, entsprechend zu reagieren, dass er nicht zurück auf das Feld fährt.

Ich lass mir da was Sinvolles einfallen, um das wieder angenehm möglich zu machen.

**Reply by acton59** [reply]
Danke
